config/audit.php

```diff
@@ -155,4 +155,16 @@
     */
 
     'console' => false,
+
+    /*
+    |--------------------------------------------------------------------------
+    | Audit Allow Saving NonScalar Values and Recursive Arrays
+    |--------------------------------------------------------------------------
+    |
+    | Allow Saving NonScalar Values and Recursive Arrays
+    |
+    */
+
+    'nonScalar' => true,
```

Comment:
Follow the configuration index convention and use `snake_case`.

Reply:
fixed